Packs/Threat_Crowd/Integrations/ThreatCrowd_v2/ThreatCrowd_v2.py

```diff
@@ -0,0 +1,299 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+from typing import Dict, Any, List, Optional, Tuple
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+VENDOR = 'Threat Crowd'
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+DEFAULT_RESOLUTION_LIMIT = 10
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    def __init__(self, base_url: str, verify: bool, proxy: bool, reliability: DBotScoreReliability, extended_data: bool):
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy)
+        self.reliability = reliability
+        self.extended_data = extended_data
+
+
+''' HELPER FUNCTIONS '''
+
+
+def handle_resolutions(resolutions: List[dict], limit: Optional[int]) -> List[dict]:
+    """ Gets a resolution section from response.
+     return a limited sorted list, desc by time.
+     Resolution section should be with following struct: [{"last_resolved": "2014-12-14", "domain": "example.com"},"""
+    resolutions = resolutions[:limit]
+    resolutions.sort(key=lambda x: x['last_resolved'], reverse=True)
+    return resolutions
+
+
+def _get_dbot_score(json_res: dict) -> Tuple[int, str]:
+    """Gets a json response and calculate the dbot score by the response. Returns both the score in code and as
+    string """
+
+    if json_res.get('response_code') == '1':
+        votes = json_res.get('votes')
+        if votes == -1:
+            return Common.DBotScore.BAD, 'BAD'
+        elif votes == 0:
+            return Common.DBotScore.SUSPICIOUS, 'SUSPICIOUS'
+        elif votes == 1:
+            return Common.DBotScore.GOOD, 'GOOD'
+
+    return Common.DBotScore.NONE, 'None'
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def ip_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    command_results: List[CommandResults] = []
+    api_url = 'ip/report/'
+    res_limit = None
+    ips = argToList(args.get('ip'))
+    for ip in ips:
+        res = client._http_request(method='GET', url_suffix=api_url, params={'ip': ip})
+        res['value'] = ip
+        score, score_str = _get_dbot_score(res)
+        dbot = Common.DBotScore(
+            ip, DBotScoreType.IP, VENDOR, score, reliability=client.reliability)
+        ip_object = Common.IP(ip, dbot)
+
+        markdown = f"### Threat crowd report for ip {ip}: \n"
+        markdown += f'DBotScore: {score_str} \n'
+
+        if not client.extended_data:
+            res_limit = DEFAULT_RESOLUTION_LIMIT
```

Comment:
In some places, we call the argument holding this value `res_limit`. In others `resolution_limit`. let's try to use consistent naming